mitodl/mitxpro
# #489 error pages 404 500

#### Pre-Flight checklist


#### What are the relevant tickets?
#489 

#### What's this PR do?
Adds 404 and 500 error pages.

#### How should this be manually tested?
Switch `DEBUG=False` in your settings and try to go to a course that does not exist. For viewing the internal server error page you will have to manually raise an exception in some handler. One way to do that is to temporarily add `raise ValueError('')` in the `handler404` method inside `mitxpro/view.py`. That way going to a non-existent course should show the 500 error page.

#### Screenshots (if appropriate)
![screencapture-xpro-odl-local-8053-courses-seed-digital-learning-500-2019-06-21-13_36_10](https://user-images.githubusercontent.com/45350418/59909765-ec449b80-9429-11e9-97f8-51164f27eaef.png)
![screencapture-xpro-odl-local-8053-courses-seed-digital-learning-500-2019-06-21-13_35_43](https://user-images.githubusercontent.com/45350418/59909775-ef3f8c00-9429-11e9-8329-876c495b3f68.png)
![screencapture-xpro-odl-local-8053-courses-seed-digital-learning-500-2019-06-21-13_34_57](https://user-images.githubusercontent.com/45350418/59909780-f1094f80-9429-11e9-9312-c3b08b66768d.png)
![screencapture-xpro-odl-local-8053-courses-seed-digital-learning-500-2019-06-21-13_34_06](https://user-images.githubusercontent.com/45350418/59909787-f4044000-9429-11e9-99e9-5a4ebfa9d4fb.png)
